demos/python_demos/speech_recognition_demo/utils/default_alphabet_example.conf

```diff
@@ -1,8 +1,7 @@
-# Each line in this file represents the Unicode codepoint (UTF-8 encoded)
-# associated with a numeric label.
-# A line that starts with # is a comment. You can escape it with \# or \\ if you wish
-# to use '#' or '\' as a label.
-
+# Each line in this file represents the Unicode string (UTF-8 encoded) associated with the position in the alphabet.
+# A line that starts with "#" is a comment. Escape the first character with "\" to use '#' or '\' as a label.
+# Whitespace sybmol can be entered as a whitespace character on a separate line or with "\w".
+\w
```

Comment:
```suggestion
# A space character can be entered as "\s".
\s
```

"Whitespace" is a general term that includes spaces, tabs and linebreaks. Here you're specifically talking about spaces.

Reply:
Let's split then this PR into two PRs: I'll keep the timer fix in this PR, and will make a new PR with example alphabet file fix.  I removed this commit from this PR (check "Files changed" tab).

Reply:
Don't know why it was needed, but sure, works for me.